app/mailers/hearing_mailer.rb

```diff
@@ -120,9 +150,31 @@ def confirmation_subject
     end
   end
 
+  def virtual_conversion_confirmation_subject
+    case recipient.title
+    when HearingEmailRecipient::RECIPIENT_TITLES[:appellant]
+      "Your Board hearing will now be a virtual hearing"
+    when HearingEmailRecipient::RECIPIENT_TITLES[:representative]
+      "Confirmation: #{appellant_name}'s Board hearing is now a virtual hearing"
+    end
+  end
+
+  def not_virtual_conversion_confirmation_subject
```

Comment:
I think i prefer this naming
```suggestion
  def convert_from_virtual_confirmation_subject
```

Reply:
Did this renaming.